sections/main-footer.liquid

```diff
@@ -2,17 +2,28 @@
 
 {% style %}
   :root  {
-    --yc-footer-background-color: {%- if section.settings.footer_background_color -%}
-      {{ section.settings.footer_background_color.hex }}
-    {%- else -%}
-      var(--yc-primary-color){%- endif -%};
+    --yc-footer-background-color: {{ section.settings.footer_background_color.hex }};
     --yc-footer-text-color: {{ section.settings.footer_text_color.hex }};
-    --yc-footer-copyright-background: {{ section.settings.footer_copyright_background.hex }};
+    --yc-footer-border-color: {{ section.settings.footer_border_color.hex }};
   }
 {% endstyle %}
 
 <footer class='yc-footer'>
   <div class="columns container">
+    {%- if section.settings.footer_logo.src -%}
+      <div class='footer-logo column'>
+        <img loading='lazy' src='{{ section.settings.footer_logo.src }}'>
+      </div>
+    {%- endif -%}
```

Comment:
if the user doesn't upload any logo we need to display the default logo of the theme